src/plugins/kibana_react/public/saved_objects/saved_object_finder.test.tsx

```diff
@@ -305,7 +347,7 @@ describe('SavedObjectsFinder', () => {
 
     expect(core.savedObjects.client.find).toHaveBeenCalledWith({
       type: ['search', 'vis'],
-      fields: ['title', 'visState'],
+      fields: ['title'],
```

Comment:
qq: since you removed `visState` as a default field, I just wonder if you're sure, that this field isn't needed in any other files than `visualize_embeddable_factory.tsx`, I didn't find one, but I couldn't rule it out completely